dali-mqtt-daemon.py

```diff
@@ -296,6 +297,14 @@ def create_mqtt_client(
             from dali.driver.hasseb import SyncHassebDALIUSBDriver
 
             dali_driver = SyncHassebDALIUSBDriver()
+            firmware_version = float(dali_driver.readFirmwareVersion())
+            if firmware_version < MIN_HASSEB_FIRMWARE_VERSION:
+                logger.error("Using a script requires newest hasseb firmware")
+                logger.error(
+                    "Please, look at https://github.com/hasseb/python-dali/tree/3dbf4af3b3770431e7351057ea328b4dbcc3a355/dali/driver/hasseb_firmware"
```

Comment:
don't link a random commit (check my comment in the previous PR review)

Reply:
Ok, but i'm not sure where we should write? Create issue on a [sde1000/python-dali](https://github.com/sde1000/python-dali) or [hasseb/python-dali](https://github.com/hasseb/python-dali)? 

Maybe we can change this link later, because I think that this take long time before hasseb write about it somewhere on wiki

Reply:
better write in hasseb repository as sde1000 does not want driver specific issues into python-dali (I tend to agree with him... I'm still bothered by our pyusbhid requirement...)

Reply:
But on hasseb/python-dali issues are closed

Reply:
best to email him, take opportunity and ping him on sde1000/python-dali/pull#78

Reply:
I ping they on this PR